frontend/gtkmm/src/TimerPreferencesPanel.hh

```diff
@@ -78,6 +78,7 @@ private:
   Gtk::CheckButton *skippable_cb;
   Gtk::CheckButton *activity_sensitive_cb;
 #ifdef HAVE_MICRO_BREAK_ACTIVITY
+  Gtk::CheckButton *quiet_cb;
```

Comment:
Though HAVE_MICRO_BREAK_ACTIVITY is basically always defined, I would put quiet_cb outside the #ifdef/endif

Reply:
done